### PR TITLE
Fix remove left margin for checkboxes

### PR DESCRIPTION
**Pull request by skanderturki**

### CODE REVIEW

File: src/foam/u2/CheckBox.js [hunk 1]
```diff
@@ -75,7 +75,7 @@ foam.CLASS({
 
   css: `
     ^ {
-      margin: 4px;
+      margin: 4px 0px 4px 0px;
```
Comment (kgrgreer) [inline src/foam/u2/CheckBox.js]: Is this this same as just margin: 4px 0;  ?

Comment (skanderturki) [reply src/foam/u2/CheckBox.js]: yes

**Review by TW80000 — CHANGES_REQUESTED** [review]
If this is to align the checkbox with the label in detail views, then we should also remove the top and bottom margin so it's consistent with the other inputs.

File: src/foam/u2/CheckBox.js [hunk 2]
```diff
@@ -75,7 +75,7 @@ foam.CLASS({
 
   css: `
     ^ {
-      margin: 4px;
+      margin: 4px 0px;
```
Comment (TW80000) [inline src/foam/u2/CheckBox.js]:
```suggestion
      margin: 0;
```

Comment (skanderturki) [reply src/foam/u2/CheckBox.js]: I tried that but the checkbox becomes too close to the label

Comment (TW80000) [reply src/foam/u2/CheckBox.js]: That's not what I'm seeing. With no margin, it's exactly the same distance from the label as text inputs, whereas with 4px it's (obviously) 4px lower than it should be.

Screen Shot 2019-10-04 at 12 48 45 PM

Comment (skanderturki) [reply src/foam/u2/CheckBox.js]: You're right, in my pc (chrome or safari) there is absolutely no space for both input and checkboxes, so you're right it's more consistent to set all margins to 0


Screen Shot 2019-10-04 at 10 37 10 AM

**Review by TW80000 — APPROVED** [review]

**Pull request merged**